packages/TelegramClient-Core.package/TCCGroupChat.class/instance/initialize.st

```diff
@@ -4,4 +4,5 @@ initialize
 	super initialize.
 	self
 		description: '';
-		members: OrderedCollection new.
\ No newline at end of file
+		members: OrderedCollection new;
+		isChannel: false.
```

Comment:
Is explicit false necessary? I think `super initialize` sets it to false by default.
(Although you could make the argument that explicit = better as well)

Reply:
we removed it here